docs/self-managed/operational-guides/multi-region/dual-region-ops.md

```diff
@@ -423,12 +422,12 @@ helm install $HELM_RELEASE_NAME camunda/camunda-platform \
 
 The following command will show the deployed pods of the newly created region.
 
-Depending on your chosen `clusterSize`, you should see that half of the amount are spawned in Zeebe brokers.
+Depending on your chosen `clusters`, you should see that half of the amount is spawned in Zeebe brokers.
```

Comment:
I'd keep it `clusterSize` as it refers to the below value in the Zeebe settings in line 427.
Otherwise, `clusters` is such an overused word in our context - Kubernetes cluster, Zeebe Cluster, Camunda cluster - that it won't be immediately visible what is actually being referred to.